src/LinAlg/PETScMatrix.h

```diff
@@ -34,7 +35,7 @@ typedef std::vector<ISVec>       ISMat;        //!< Index set matrix
   linear systems of equations.
 */
 
-class PETScVector : public SystemVector
+class PETScVector : public StdVector
```

Comment:
Så nå blir altså PETScVector en sub-klasse av StdVector. Det betyr vel at det ikke er andre under-klasser av SystemVector enn nettop StdVector. Da kan man vurdere å slå sammen StdVector og SystemVector for å gjøre hierarkiet enklere.